src/draw/vg_lite/lv_draw_vg_lite_img.c

```diff
@@ -159,14 +212,14 @@ void lv_draw_vg_lite_img(lv_draw_task_t * t, const lv_draw_image_dsc_t * dsc,
             &src_buf,
             &matrix,
             blend,
-            VG_LITE_PATTERN_COLOR,
+            dsc->tile ? VG_LITE_PATTERN_COLOR : VG_LITE_PATTERN_REPEAT,
```

Comment:
Shouldn't it be this?

```suggestion
            dsc->tile ? VG_LITE_PATTERN_REPEAT : VG_LITE_PATTERN_COLOR,
```

Reply:
Nice catch